Wiki/Testnet.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Gridcoin Testnet
+layout: wiki
+---
+
+# Introduction
+
+Testnet (test network) is a separate blockchain meant for testing new features 
+of Gridcoin. It helps catch issue before a release so they don't end up in the
+final versions. The coins on testnet don't have actual value since they are just 
+for testing purposes. Want to help test? See the [How To Help Section](#how-to-help) 
+for more info
+
+Secondarily it also can be used to help test scripts or services that interact 
+with Gridcoin without using real coins. This does not guarantee that your
+program will work on the main network (mainnet), but it can be helpful. Make sure 
+to note some of differences in the [Different To Mainnet Section](#difference-to-mainnet)
```

Comment:
these sections have different titles and headers.  Suggest to pick one

Reply:
If you are referring to different vs difference, that's been fixed.

If you are referring to the links being lowercase and hyphenated, that's because kramdown generates them ignoring capitalization and hyphenating spaces. I still want to leave the headers on the page in uppercase and with spaces

Reply:
The former :)